docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+# Default Reference Types
+
+We've mentioned throughout this tutorial what the default reference capabilities are for types and classes. However, there's a good chance those defaults passed over your head while you were focused on understanding capabilities themselves.
+
+First, there is nothing saying you have to use default capabilities. Especially as you start writing your first Pony programs, you'll probably find it very educational to explicitly specify the capabilities for all your classes, actors, functions, parameters and fields. However, it's not _writing_ code that makes default capabilities confusing; it's _reading_ that code. Hopefully keeping this summary open will help you navigate pony source code until you've become used to the defaults!
```

Comment:
This says "First," but there's no "Second". Were you intending to add another point? If not, I'd like to remove the "First," but I'm not sure if in the process of writing if you had some additional points you were thinking of making and then didn't while working out other parts.

Reply:
I meant it more as "Before we begin", but if I phrase it that way the whole paragraph falls apart. I'll stitch it together differently when I push changes.